python/pylibraft/pylibraft/test/test_kmeans.py

```diff
@@ -88,3 +88,31 @@ def test_compute_new_centroids(
     actual_centers = new_centroids_device.copy_to_host()
 
     assert np.allclose(expected_centers, actual_centers, rtol=1e-6)
+
+
+@pytest.mark.parametrize("n_rows", [8])
```

Comment:
Might be a good idea to test this with a slightly larger number like 100.

Reply:
nice catch! I had reduced it for debugging - 

its back at 100 in the last commit